futures-util/src/compat/compat.rs

```diff
@@ -1,20 +1,20 @@
-/// Converts a futures 0.3 `TryFuture` into a futures 0.1 `Future`
-/// and vice versa.
+/// Converts a futures 0.3 `TryFuture`, `TryStream` or `Sink` into a futures 0.1
+/// `Future` and vice versa.
```

Comment:
These should be links

Reply:
Ah never mind. Let's do the docs stuff in another PR

Reply:
Yeah, I started doing it then decided to just leave it for another pass.